cmd/pd-server/main.go

```diff
@@ -88,7 +88,7 @@ func main() {
 		syscall.SIGTERM,
 		syscall.SIGQUIT)
 
-	if err := svr.Run(); err != nil {
+	if err := svr.Run(sc); err != nil {
```

Comment:
why not pass a Context? So we can cancel the running.

Reply:
Good idea.